lib/ap/plan9/unlink.c

```diff
@@ -70,7 +70,7 @@ unlink(const char *path)
 					nn = 0;
 				seek(fd, nn, 0);
 				_DUP(fd, i);
-				close(fd);
+				_CLOSE(fd);
```

Comment:
Aligning with original NIX source, in which there are certain sources under ap/plan9 what are using open/close syscalls and not the functions provided in plan9/open.c or plan9/close.c. Look at the differences. Note that posix programs will only use open/close functions, also rest of ap sources out of plan9 dir.